### PR TITLE
Update documentation to make the possible settings for state_characteristic more searchable

### PR DESCRIPTION
**Pull request by YogoGit**
## Proposed change

The setting for state_characteristic are not easily determined without context, so provide a bit more context that should allow both humans and search engines the ability to view all the possible combinations for state_chacteristic for both sensor and binary_sensor.

## Type of change

- [X] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant.io/issues/26586
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/statistics.markdown [hunk 1]
```diff
@@ -34,7 +34,7 @@ The following statistical characteristics are available. Pay close attention to
 
 ### Numeric Source Sensor
 
-The following characteristics are supported for `sensor` source sensors:
+The following are supported for `sensor` source sensors for `state_characteristic`:
```
Comment (ThomDietrich) [inline source/_integrations/statistics.markdown]: Could you please get rid of the double "for" ?

File: source/_integrations/statistics.markdown [hunk 2]
```diff
@@ -67,7 +67,7 @@ The following characteristics are supported for `sensor` source sensors:
 
 ### Binary Source Sensor
 
-The following characteristic are supported for `binary_sensor` source sensors:
+The following are supported for `binary_sensor` source sensors for `state_characteristic:
```
Comment (ThomDietrich) [inline source/_integrations/statistics.markdown]: Same as above

File: source/_integrations/statistics.markdown [hunk 3]
```diff
@@ -129,7 +129,7 @@ name:
   default: Stats
   type: string
 state_characteristic:
-  description: The characteristic that should be used as the state of the statistics sensor (see table above).
+  description: The characteristic that should be used as the state of the statistics sensor (see tables above for each sensor type).
```
Comment (ThomDietrich) [inline source/_integrations/statistics.markdown]: "tables" agreed, but the rest can be removed I believe

**Comment by ThomDietrich** [comment]
Additionally, let's mention the parameter name in line 33 as well.

**Comment by YogoGit** [comment]
Updated per comments (rebased to have a single commit).

**Review by YogoGit — COMMENTED** [review]
Not a fan of how this reads.  Suggestions welcome.

File: source/_integrations/statistics.markdown [hunk 4]
```diff
@@ -30,11 +30,11 @@ The `statistics` integration is different to [Long-term Statistics](https://deve
 
 ## Characteristics
 
-The following statistical characteristics are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
+The following statistical characteristics (`state_characteristic`) are available. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```
Comment (YogoGit) [inline source/_integrations/statistics.markdown]: Not a fan of how this flows.  Suggestions welcome.

File: source/_integrations/statistics.markdown [hunk 5]
```diff
@@ -30,11 +30,11 @@ The `statistics` integration is different to [Long-term Statistics](https://deve
...
 
 ### Numeric Source Sensor
 
-The following characteristics are supported for `sensor` source sensors:
+The following are supported for `sensor` source sensors `state_characteristic`:
```
Comment (mrand) [inline source/_integrations/statistics.markdown:37]: I think this one flows worse (same for the binary further below).   Can the word "source" be eliminated?  I know the following isn't perfect, but maybe something like:
```
The following `state_characteristic`s are supported by the `sensor` type.
```

File: source/_integrations/statistics.markdown (hunk 4 above)
Comment (mrand) [inline source/_integrations/statistics.markdown]:
```suggestion
The following statistical characteristics (`state_characteristic`) are available. Pay close attention to the correct configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```
While maybe not perfect, it seems reasonable to me.

Comment (ThomDietrich) [reply source/_integrations/statistics.markdown]: Agree with this.

File: source/_integrations/statistics.markdown (hunk 5 above)
Comment (ThomDietrich) [reply source/_integrations/statistics.markdown:37]: This documentation document talks about the "source sensor" or "source entity" consistently, to establish a clear difference between the created statistics sensor, and the underlying source sensor it is derived from.

```suggestion
The following options for `state_characteristic` are supported for source entities of the `sensor` domain:
```

File: source/_integrations/statistics.markdown (hunk 4 above)
Comment (ThomDietrich) [reply source/_integrations/statistics.markdown]:
```suggestion
The following options for the configuration parameter `state_characteristic` are available as statistical characteristics. Pay close attention to the right configuration of `sampling_size` and/or `max_age`, as most characteristics are directly influenced by these settings.
```

File: source/_integrations/statistics.markdown (hunk 5 above)
Comment (YogoGit) [reply source/_integrations/statistics.markdown:37]: I'm getting mixed signals here.  How should I proceed?

File: source/_integrations/statistics.markdown (hunk 4 above)
Comment (YogoGit) [reply source/_integrations/statistics.markdown]: I used a mix of both the suggestions (changing 'right' -> 'correct'), but using the same sentence structure as above.

**Review by frenck — APPROVED** [review]
Thanks, @YogoGit 👍 

../Frenck

**Comment by ThomDietrich** [comment]
Thanks @YogoGit for your contribution

**Pull request merged**